src/clock/stringPrinting.c

```diff
@@ -0,0 +1,44 @@
+#include "stringPrinting.h"
+
+#include <SDL/SDL.h>
+
+#include "font_drawing.h"
+
+#define SDL_BLACK_COLOR 0
+
+uint16_t GetSDLColor(SDL_Surface* sdl_screen, uint8_t red, uint8_t green, uint8_t blue)
+{
+	return SDL_MapRGB(sdl_screen->format, red, green, blue);
+}
+
+uint16_t GetSDLWhiteColor(SDL_Surface* sdl_screen)
+{
+	return GetSDLColor(sdl_screen, 255, 255, 255);
+}
+
+uint16_t GetSDLDarkGrayColor(SDL_Surface* sdl_screen)
+{
+	return GetSDLColor(sdl_screen, 70, 70, 70);
+}
```

Comment:
I would just have declared the colors statically:
```c
static SDL_Color color_white = {255, 255, 255};
static SDL_Color color_gray = {70, 70, 70};
```

So much boilerplate!

Reply:
done